llvm/lib/Support/Process.cpp

```diff
@@ -102,10 +102,31 @@ bool Process::AreCoreFilesPrevented() { return coreFilesPrevented; }
     ::exit(RetCode);
 }
 
+#if defined(_WIN32) || defined(__CYGWIN__)
```

Comment:
Supporting what @mstorsjo said, this should be done in opposite direction. Process.cpp is mostly considered as "target-neutral"